cves/kernel/CVE-2021-3178.yml

```diff
@@ -320,8 +335,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: J. Bruce Fields and Chuck Lever signed off on the commit.
```

Comment:
Mentioned this in another PR but I'm not sure if we should mention names directly? I ping'ed professor here about it https://github.com/VulnerabilityHistoryProject/vulnerabilities/pull/42#discussion_r1002561717